migrations/1688126055_closedJobs.ts

```diff
@@ -0,0 +1,61 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+
+  console.log("Creating new fields/fieldsets");
+
+  console.log(
+    'Create Boolean field "Is closed?" (`closed`) in model "Job" (`job`)'
+  );
+  newFields["10413779"] = await client.fields.create("38231", {
+    label: "Is closed?",
+    field_type: "boolean",
+    api_key: "closed",
```

Comment:
Thoughts on naming this `is_closed` think it's common naming pattern for booleans?